pkg/descriptions.json

```diff
@@ -1962,6 +1962,13 @@
       "System.Text.Encodings.Web.JavaScriptEncoder"
     ]
   },
+  {
+    "Name": "System.Text.Json",
+    "Description": "Low allocation UTF-8 JSON reader/writer.",
+    "CommonTypes": [
+      "System.Text.Json.JsonUtf8Reader"
```

Comment:
I thought we were going to go with the alternate name of Utf8JsonReader to be consistent with other types like Utf8String, no?

Reply:
No, from the API review: https://github.com/dotnet/apireviews/pull/73/files#diff-244c336af7586c3fec32cb2d485d9dc7R63

**Also, related discussion:**
https://github.com/dotnet/corefxlab/pull/2577#discussion_r229773039

We can re-visit the name if you feel strongly about it.

Reply:
Despite being in the camp that suggested moving the "Utf8" to after "Json", I keep reading this now as some sort of JSON-ified "UTF-8 Reader".  So maybe we should take another shot at the name.  Utf8JsonReader or JsonReaderUtf8 being the obvious remaining candidates.  "JsonReaderUtf8" sorts better, "Utf8JsonReader" seems more Englishy.

Reply:
I'm not worried about sorting: it's all in the Json namespace, and IntelliSense handles such cases well IMO.   So from my perspective if we're not going to go with JsonUtf8Reader, it should be Utf8JsonReader.

Reply:
For what it's worth `JsonUtf8Reader` sounds like it's `some sort of JSON-ified "UTF-8 Reader"` to me as well. Either of those alternatives sound better to me.